### ISSUE TITLE
The realization of the Dnc in paper.

### ISSUE THREAD

**Issue by lmTEDug** [vrt1shjwlkr/NDSS21-Model-Poisoning#1]
Can I have the relevant code for the defense mechanism, please?

**Comment by vrt1shjwlkr** [comment]
Hi @lmTEDug, please allow me a few days to upload an implementation of DnC. Thanks!

**Comment by lmTEDug** [comment]
> Hi @lmTEDug, please allow me a few days to upload an implementation of DnC. Thanks!

Of course, and I will always keep an eye on it in the next few days. Look forward to your upload and thank you sincerely!

**Comment by lmTEDug** [comment]
> Hi @lmTEDug, please allow me a few days to upload an implementation of DnC. Thanks!

Hello, sorry to bother you again, could you please let me know a specific upload time?

**Comment by lishenghui** [comment]
> > Hi @lmTEDug, please allow me a few days to upload an implementation of DnC. Thanks!
> 
> Hello, sorry to bother you again, could you please let me know a specific upload time?

Hi, I have implemented DnC in my framework, [Blades](https://github.com/bladesteam/blades/blob/master/src/blades/aggregators/dnc.py).  Feel free to use it!

**Comment by KaiyuanZh** [comment]
> Hi @lmTEDug, please allow me a few days to upload an implementation of DnC. Thanks!

Dear authors,

Thank you so much for the nice work and releasing the code! I am also interested in the defense and planning a follow-up work. May I kindly ask if it is possible to release the defense code?

Best,
Kaiyuan Zhang

**Comment by Enrique-Marmol** [comment]
Hello, is the DnC method implemented in the code?
